convert.go

```diff
@@ -134,6 +133,12 @@ func ToString(arg any, timeFormat ...string) string {
 	case fmt.Stringer:
 		return v.String()
 	default:
+		// Check if the type is a pointer by using reflection
+		rv := reflect.ValueOf(arg)
+		if rv.Kind() == reflect.Ptr && !rv.IsNil() {
+			// Dereference the pointer and recursively call ToString
+			return ToString(rv.Elem().Interface(), timeFormat...)
+		}
 		return ""
```

Comment:
Here we can use fmt.Sprint for the worst-scenario.

Reply:
Added sprint for types not being handled, we still need to handle pointer dereferencing with our recursive logic.
But yeah also had to update our test cases since unknown structs are handled now ~

```go
ToString({struct{ Name string }{"John"}) // "{John}"
```